docs/examples/exporting_models_json.py

```diff
@@ -1,4 +1,5 @@
 from datetime import datetime, timedelta
+import pendulum
```

Comment:
I think too much extra confusion for those who haven't heard of pendulum to introduce another package for one test. Either find an example in another package we use or create a synthetic example.

You can add a note about pendulum, asyncpg etc. in text.